scheds/rust/scx_layered/src/log_recorder.rs

```diff
@@ -0,0 +1,294 @@
+// Copyright (c) Meta Platforms, Inc. and affiliates.
+// Copyright (c) Netflix, Inc.
+// Author: Jose Fernandez
+
```

Comment:
A significant portion of this file was taken from log reporting logic in main.rs

Please advice whether this header correctly captures this or if you'd like me to do any modifications.